config/1.0.0/earthdata_varinfo_configuration_schema.json

```diff
@@ -317,40 +317,6 @@
             "description": "A list of metadata attributes with their names and values.",
             "$ref": "#/$defs/AttributesItemType"
           }
-        },
-        "Applicability_Group": {
-          "description": "A more stringent applicability role that will also include a variable path, allowing for overwriting or supplementing the metadata attribute of a subset of variables within a granule.",
-          "type": "array",
-          "items": {
-            "$ref": "#/$defs/ApplicabilityGroupItemType"
-          }
-        }
-      },
-      "additionalProperties": false,
-      "anyOf": [{
```

Comment:
So this bit, with two options for the attributes in the override/supplement is now being replaced by [this line](https://github.com/nasa/earthdata-varinfo/pull/33/files#diff-cb2e38ad70c1243f56366dec874a890b06e7be15cf31b54e8ad770ba93a8cc98R323).